scripts/zones/Windurst_Waters/npcs/Leepe-Hoppe.lua

```diff
@@ -201,6 +225,17 @@ function onEventFinish(player,csid,option)
         player:setCharVar("MissionStatus",1);
     elseif (csid == 742) then
         finishMissionTimeline(player,3,csid,option);
+
+    -- Tuning In
+    elseif csid == 884 then
+        player:addQuest(WINDURST,tpz.quest.id.windurst.TUNING_IN)
+
+    elseif csid == 886 then
```

Comment:
If you make this line elseif csid ==886 and npcUtil.compleQuest.... then
    Player:tradeComplete()
End. 

Then it will only take the trade if it in facts completes the quest

Reply:
Good call!